src/view/screens/Home.tsx

```diff
@@ -152,12 +159,15 @@ function HomeScreenReady({
   const onPageSelecting = React.useCallback(
     (index: number, reason: LogEvents['home:feedDisplayed']['reason']) => {
       const feed = allFeeds[index]
-      logEvent('home:feedDisplayed', {
-        index,
-        feedType: feed.split('|')[0],
-        feedUrl: feed,
-        reason,
-      })
+
+      if (feed) {
```

Comment:
Can this ever be empty? Seems like we'd expect the index during selection to always be in bounds. Has something changed here?

Reply:
I don't remember why I added this but yeah I don't think it's needed 👍